### PR TITLE
Add release docs 1.3.0

### PR DESCRIPTION
**Pull request by angelbarrera92**
Hi team. 

I wanted to release a new monitoring module version. I increased a minor version because we include new features (goldpinger). 

Once this PR got merged, it's needed to create a git tag in the master branch.

```bash
$ git pull origin master
$ git tag v1.3.0
$ git push --tags
```

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ specific dependencies please visit the single package's documentation:
 | v1.0.0                              |                    | :white_check_mark: |                    |
 | v1.1.0                              | :white_check_mark: | :white_check_mark: | :x:                |
 | v1.2.0                              | :white_check_mark: | :white_check_mark: | :x:                |
+| v1.3.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (phisco) [inline README.md]: just to be clear, which are the reasons to bump minor and not patch version?

Comment (phisco) [reply README.md]: probably these reasons should be clearly stated in the changelog, e.g. dividing between breaking/non breaking changes.

Comment (angelbarrera92) [reply README.md]: As i wrote in the description of the PR:

> I increased a minor version because we include new features (goldpinger).

Comment (phisco) [reply README.md]: ops, sorry, you are right, LGTM

**Comment by angelbarrera92** [comment]
Tag created ;)

**Pull request merged**